src/olympia/lib/settings_base.py

```diff
@@ -752,8 +752,6 @@ def get_db_config(environ_var, atomic_requests=True):
 # Email goes to the console by default.  s/console/smtp/ for regular delivery
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-# Please use all lowercase for the QA allow list.
-EMAIL_QA_ALLOW_LIST = env.list('EMAIL_QA_ALLOW_LIST', default=())
```

Comment:
We could have a `RunPython` migration that (attempts to) prefill the `Group` with the current values?

Reply:
From a look at the current data it seems all worthless